feedback-module/src/components/Module.js

```diff
@@ -122,9 +119,18 @@ function Module({ pagetitle, endpoint, dir }) {
       pageTitleAsScreen(currentPageTitle);
       pageChange(currentPageTitle, nextPageTitle);
 
-      sendFormData(screen.formID),
-        setScreen(SCREENS[nextScreen]),
-        setCheckboxError(false);
+      if (requestInfo(screen.formID)) {
```

Comment:
Why do you need this if statement here?

Reply:
Hypothetically, if the configuration is correct and and the formID matches one of the if or else if checks in requestInfo, we wouldn't need it. This is just a safeguard to handle the case where a system administrator configures an invalid formID and there isn't anything returned by requestInfo